grafana/grafana.go

```diff
@@ -0,0 +1,222 @@
+package grafana
+
+import (
+	"github.com/prismatik/jabba"
+)
+
+func Go() {
+	jabba.RunOrDie("wget", "https://grafanarel.s3.amazonaws.com/builds/grafana_3.1.1-1470047149_amd64.deb")
+	jabba.RunOrDie("sudo", "apt-get", "install", "-y", "adduser", "libfontconfig")
+	jabba.RunOrDie("sudo", "dpkg", "-i", "grafana_3.1.1-1470047149_amd64.deb")
+	jabba.RunOrDie("echo", "'deb https://packagecloud.io/grafana/stable/debian/ wheezy main'", ">", "/etc/apt/sources.list")
+	jabba.RunOrDie("curl", "https://packagecloud.io/gpg.key", "|", "sudo", "apt-key", "add", "-")
+	jabba.RunOrDie("sudo", "apt-get", "update")
+	jabba.RunOrDie("sudo", "apt-get", "install", "-y", "grafana")
```

Comment:
We seem to be installing grafana twice. Once via wgetting the deb and then again via the apt repository.